CesiumIonClient/include/CesiumIonClient/Connection.h

```diff
@@ -103,9 +104,20 @@ class CESIUMASYNC_API Connection {
       const std::string& redirectPath,
       const std::vector<std::string>& scopes,
       std::function<void(const std::string&)>&& openUrlCallback,
+      const CesiumIonClient::ApplicationData& appData,
       const std::string& ionApiUrl = "https://api.cesium.com/",
       const std::string& ionAuthorizeUrl = "https://ion.cesium.com/oauth");
 
+  /**
+   * @brief Retrieves information about the ion API server.
+   *
+   * @return A future that resolves to the application information.
+   */
+  static CesiumAsync::Future<Response<ApplicationData>> appData(
```

Comment:
Kinda think the verb form "getAppData" would be more descriptive.

Reply:
This would make sense, but I wanted to keep the naming consistent with the rest of the methods. In this class, all the get* methods return private members of the class, whereas all the methods calling endpoints are named with that endpoint name (`me`, `assets`, `defaults`, etc).